syntax = "proto3";

package picoradar;

import "player_data.proto";

// --- 鉴权消息 ---
message AuthRequest {
  string token = 1; // 预共享的秘密令牌
  string player_id = 2; // 客户端的玩家ID
}

// --- 客户端 -> 服务端 ---
message ClientToServer {
  oneof message_type {
    AuthRequest auth_request = 1;
    PlayerData player_data = 2;
  }
} 